apps/demos/Demos/ContextMenu/Scrolling/Angular/app/app.service.ts

```diff
@@ -2,1243 +2,344 @@ import { Injectable } from '@angular/core';
 
 export class ContextMenuItems {
   text: string;
-
-  items: ContextMenuItems[];
+  
```

Comment:
Let's revert this change too

Reply:
[c076b11](https://github.com/DevExpress/DevExtreme/pull/27367/commits/c076b116b3302bdc6cb4017afe1948f85476031b)